sycl/test/esimd/memory_properties.cpp

```diff
@@ -0,0 +1,101 @@
+// RUN: %clangxx -O0 -fsycl -fsycl-device-only -Xclang -emit-llvm %s -o %t
+// RUN: sycl-post-link -split-esimd -lower-esimd -O0 -S %t -o %t.table
+// RUN: FileCheck %s -input-file=%t_esimd_0.ll --check-prefixes=CHECK
```

Comment:
Nit: I dont think you need `--check-prefixes=CHECK`, it should be the default

Reply:
Yes, thank you. It was copied from other test that tested compilation in 2 modes with and without force-stateless-mem
I will remove `--check-prefixes=CHECK`.
We'll need `--check-prefixes' when start adding accessor-based API to this test.